Game/Source/TestSuite/Tests/TestStaticComponentReplication.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "TestStaticComponentReplication.h"
+
+#include "GameFramework/GameModeBase.h"
+#include "UnrealNetwork.h"
+
+UTestComponent::UTestComponent()
+{
+	PrimaryComponentTick.bCanEverTick = true;
+	TestProperty = 42.f;
+}
+
+void UTestComponent::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+	DOREPLIFETIME_CONDITION(UTestComponent, TestProperty, COND_None);
+}
+
+void UTestComponent::OnRep_TestProperty()
+{
+	auto Owner = Cast<ATestStaticComponentReplication>(GetOwner());
+	check(Owner);
+	Owner->Server_ReportReplication(TestProperty);
+}
+
+ATestStaticComponentReplication::ATestStaticComponentReplication()
+{
+	TestName = TEXT("Static component replication");
+	TestComponent = CreateDefaultSubobject<UTestComponent>(TEXT("TestComponent"));
+	check(TestComponent);
+	TestComponent->SetIsReplicated(true);
+}
+
+void ATestStaticComponentReplication::GetLifetimeReplicatedProps(TArray< FLifetimeProperty > & OutLifetimeProps) const
```

Comment:
Do we intend to add properties to this class in the future? If not, we don't need to override this function.